impit-node/index.d.ts

```diff
@@ -10,7 +10,7 @@ export type ImpitWrapper = Impit
 export declare class ImpitResponse {
   status: number
   statusText: string
-  headers: Record<string, string>
+  headers: Headers
```

Comment:
This change is theoretically breaking for anyone reading the headers like `response.headers[headerName]`. In our codebase, this is done e.g. in Crawlee's [`impit-client`](https://github.com/apify/crawlee/blob/668637d4ce4913fd7e3a7dbecb59192f1d15fd56/packages/impit-client/src/index.ts#L115). 

We either need to pin the `impit` version there, or make the `impit-client` implementation ready for either "syntax".